internal/precache/workload/pullImages.go

```diff
@@ -158,9 +172,18 @@ func PullImages(precacheSpec []string, authFile string) *precache.Progress {
 func ValidatePrecache(status *precache.Progress, bestEffort bool) error {
 	// Check pre-caching execution status
 	if status.Failed != 0 {
+		var imagesFound []string
 		log.Info("Failed to pre-cache the following images:")
 		for _, image := range status.FailedPullList {
-			log.Info(image)
+			if strings.Contains(image, "@sha:") && podmanImgExists(image) {
+				log.Infof("%s, but found locally after downloading other images", image)
+				imagesFound = append(imagesFound, image)
+			} else {
+				log.Info(image)
+			}
+		}
+		if len(imagesFound) == status.Failed {
```

Comment:
I assume `status.Failed` is a count of the failed images? Not obvious from the field name, so a comment here would be good, too